apicast/src/policy.lua

```diff
@@ -0,0 +1,42 @@
+local _M = {
+    PHASES = {
+        'init', 'init_worker',
+        'rewrite', 'access', 'balancer',
+        'header_filter', 'body_filter',
+        'post_action',  'log'
+    }
+}
+
+local setmetatable = setmetatable
+local ipairs = ipairs
+
+local noop = function() end
+
+--- Initialize new policy
+-- Returns a new policy that you can extend however you want.
+-- @tparam string name Name of the new policy.
+-- @tparam string version Version of the new policy. Default value is 0.0
```

Comment:
Currently, the code does not set a default value.
Do you see any reason to set 0.0 instead of nil?

Reply:
I think it would be nice to be able to assume there is always a version.